lib/serverless/event-sources.json

```diff
@@ -21,7 +21,7 @@
             "httpMethod",
             "path",
             "requestContext",
-            "resource"
+            "requestContext.stage"
```

Comment:
I see we've removed one key for matching that does not exist in v2 and replaced with a key that exists in both.